README.md

```diff
@@ -2,7 +2,7 @@
 The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin (Unreal documentation)](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
 
 The GDK offers:
-* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one, enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
+* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.
```

Comment:
Original:
* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.

->
* Wondering if we say "Unreal game server" it suggests we are on Unreal's servers. We have made a distinction between Unreal servers and SpatialOS servers in the documentation (out of necessity because of the switch).
* "enables usage " is a bit awkward.
* "in a single instance" isn't clear what we mean? A single instance of what? When I get an idea, I can suggest some alternative text.

Also, this text is the same as the welcome/index page - so need to synch up when we have the final copy here.